src/app/Food.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package app;
+
+import javafx.scene.control.DatePicker;
+
+/**
+ *
+ * @author aswecker
+ */
+public class Food {
+    private String foodName;
+    private int servingSize;
+    private DatePicker foodDate;
+    private int id;
+    private String foodGroup;   
+    
+    /**
```

Comment:
Please add a brief description next to each @param <variable name> stating what it is (even if obvious). It doesn't need to be elaborate.